routes/lib/usertools.js

```diff
@@ -244,6 +244,35 @@ class UserTools {
 			});
 		}
 	}
+
+	generateUsername(algorithm, length) {
+		switch (algorithm) {
+		  case 'alphanumeric':
+			return this.randomString(length, 'a#');
+			break;
+		  case 'base58':
+			return this.randomString(length, 'b*');
+			break;
+		  case 'letters':
+		  default: 
+			return this.randomString(length, 'a');
+			break;
+		}
+	  }
+	  
+	  randomString(length, chars) {
```

Comment:
Can you check https://learnmeabitcoin.com/technical/keys/base58/ ?. Seems that there are still some chars to be excluded to align to base58